Test/ast/method.dfy

```diff
@@ -0,0 +1,6 @@
+// RUN: %baredafny verify --use-basename-for-filename "%s" > "%t"
+// RUN: ! %baredafny run --use-basename-for-filename "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Foo()
+method {:axiom} Bar()
```

Comment:
{:axiom} is just about assuming post-conditions without proof, so couldn't we allow bodiless methods with no post-conditions?

For the record (for later) I think I'd like to support attaching `{:axiom}` to individual `ensures` clauses instead of the top-level declaration. There are two reasons:
1. The term "axiom" is supposed to mean "assume this is true without proof". So "this post-condition is an axiom" is much more natural than "this method is an axiom" (when in this case there's actually nothing being assumed at all).
2. It would allow you to make individual `ensures` clauses axioms while still proving others, which could be useful when methods DO have bodies.

Reply:
I realize this is an orthogonal improvement to the feature you are just emitting warnings about, but if it's quick to implement it means fewer false-positive warnings so it's worth doing if possible.

Reply:
Nice suggestion! I will make this change